src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final int CAPACITY = 100;
+
     public static void main(String[] args) {
+        printWelcomeMessage();
+
+        Task[] tasks = new Task[CAPACITY];
+        int counter = 0;
+
+        while(true) {
+            String line;
+            Scanner input = new Scanner(System.in);
+            line = input.nextLine();
+
+            if (line.equalsIgnoreCase("bye")) {
+                printByeMessage();
+                break;
+            } else if (line.equalsIgnoreCase("list")) {
+                System.out.println("____________________________________________________________");
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < counter; i++){
+                    System.out.println(i+1 + ". " + tasks[i].toString());
+                }
+                System.out.println("____________________________________________________________");
+            } else if (line.startsWith("todo")) {
+                line = line.replaceAll("todo ", "");
```

Comment:
May cause bugs if there are multiple todo's within the user input (e.g. "todo finish methods for todo deadline event")